### ISSUE TITLE
Bug: Pressing enter when confirming characters in Japanese also sends chat message

### ISSUE THREAD

**Issue by slmjkdbtl** [felixrieseberg/clippy#12]
In some language input methods like Japanese "Enter" is used when confirming characters, they should not trigger a message. This bug makes it very difficult to write a full sentence in Japanese.

A fix might look something like this:

```js
let isComposing = false;

const textbox = document.getElementById("textbox-id");

textbox.addEventListener("compositionstart", () => {
  isComposing = true;
});

textbox.addEventListener("compositionend", () => {
  isComposing = false;
});

textbox.addEventListener("keydown", (event) => {
  if (event.key === "Enter" && !isComposing) {
    console.log("Enter pressed and not composing");
  }
});
```
